app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -40,6 +40,7 @@ def perform(submission_id)
           response_handler(response)
         end
         send_rrd_completed_notification(submission) if submission.rrd_job_selector.rrd_applicable?
+        submission.send_rrd_to_mas_email if submission.rrd_to_mas_disability?
```

Comment:
Ah, I see what you're saying. In that case, maybe rename `rrd_to_mas_disability?` to `forward_to_mas?` and `send_rrd_to_mas_email` to `send_to_mas_email`. The `rrd` throws me off. Since these claims are not `rrd_applicable?`, then let's remove `rrd` from the method names. Should also add in comments what `mas` means.

Reply:
@yoomlam yep, good point, that's my bad: rrd in the name is confusing! Okay, will work on better naming. Tyvm.